exchange_calendars/calendar_utils.py

```diff
@@ -350,13 +373,48 @@ def resolve_alias(self, name):
 
         return name
 
+    def aliases_to_names(self) -> dict[str, str]:
+        """Return dictionary mapping aliases to canonical names.
+
+        Returns
+        -------
+        dict of {str, str}
+            Dictionary mapping aliases to canoncial name of corresponding
+            calendar.
+
+        See Also
+        --------
+        resolve_alias : Resolve single alias to a canonical name.
+        names_to_aliases : Mapping of cononcial names to aliases.
+        """
+        return {alias: self.resolve_alias(alias) for alias in self._aliases}
+
+    def names_to_aliases(self) -> dict[str, list[str]]:
+        """Return mapping of canonical calendar names to associated aliases.
+
+        Returns
+        -------
+        dict of {str, list of str}
+            Dictionary mapping canonical calendar names to any associated
+            aliases.
+
+        See Also
+        --------
+        aliases_to_names : Mapping of aliases to canoncial names.
+        """
+        names = self.get_calendar_names(include_aliases=False)
+        dic = {name: [] for name in names}
```

Comment:
```suggestion
        dic = defaultdict(list)
```

Reply:
(though this also needs a `from collections import defaultdict`

Reply:
Thanks for reviewing.

I did it this way to include all canonical names as keys and explicitely show which calendars have no aliases (using defaultdict would have excluded calendars without aliases). I've used an empty list to indicate 'no aliases' in order to keep the same type for all values.

The method has no internal clients, rather it's intended simply to allow users to query aliases, hence I went with the more explicit approach.

I thought I'd let you consider my comment before making the requested change. If you would prefer that the method only returns those names with aliases, just say the word and I'll revise that last commit.

Reply:
Ah fair point,